src/pretalx/submission/models/submission.py

```diff
@@ -288,38 +297,71 @@ def _set_state(self, new_state, force=False, person=None):
                 )
             )
 
-    def make_submitted(self, person=None, force=False, orga=False):
-        self._set_state(SubmissionStates.SUBMITTED, force, person=person)
+    def update_TalkSlots(self, available_count_old=None):
```

Comment:
Please follow the PEP8 coding style. This would make this method name "update_talk_slots".